apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx

```diff
@@ -337,7 +336,7 @@ export const ReplaceConfirmationModal = ({ isOpen }: Props) => {
           </button>
           <button
             onClick={handleContinueImport}
-            className="h-[38px] rounded bg-controls-accent px-3 hover:bg-controls-accent-hover"
+            className="button button-primary h-[38px] rounded bg-controls-accent px-3 hover:bg-controls-accent-hover"
```

Comment:
Are you sure that other old classes are required here?
`h-[38px] rounded bg-controls-accent px-3 hover:bg-controls-accent-hover`

Reply:
I didn’t try to touch the old codes, I just added what was in the task, if necessary I can look

Reply:
`px-3` and `rounded` are optional. They are already present in the .button class. Could you please remove them?

Reply:
Yes sure, Thank you for the clarification

Reply:
@Derikyan ,
![image](https://github.com/epam/ai-dial-chat/assets/20163971/fe2633d9-af9b-4b82-bca7-5a1260d0bab2)

Reply:
"button button-primary h-[38px]" should stay there, thanks